### PR TITLE
Fix issues

### PR DESCRIPTION
**Pull request by alexkeating**
## Changes

Fix caip issue

## Packages Added
"@ceramicnetwork/3id-did-resolver": "^2.0.1-rc.4",
"@ceramicnetwork/http-client": "2.0.1-rc.4",
"@ceramicnetwork/stream-caip10-link": "2.0.1-rc.2",
"@ceramicnetwork/stream-tile": "2.1.0-rc.3",
"@gnosis.pm/safe-apps-web3modal": "^12.0.0",

## Checks

Before making your PR, please check the following:

- [ ] Critical lint errors are resolved
- [ ] App runs locally
- [ ] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: libs/dao-data/src/Profile.ts [hunk 1]
```diff
@@ -66,3 +77,59 @@ export default class Profile {
     return await provider.lookupAddress(address);
   }
 }
+
+const profilModel = {
```
Comment (skuhlmann) [inline libs/dao-data/src/Profile.ts:81]: Do we need this here?

Comment (alexkeating) [reply libs/dao-data/src/Profile.ts:81]: We can remove it. It has the metadata of for the hash we use above. I don't mind either way

File: libs/dao-data/src/Profile.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
-import { Core } from '@self.id/core';
+// import { Core } from '@self.id/core';
```
Comment (jordanlesich) [inline libs/dao-data/src/Profile.ts:1]: comment

File: libs/dao-data/src/Profile.ts [hunk 3]
```diff
@@ -25,12 +27,11 @@ type AccountProfile = {
 const CERAMIC_NETWORK = process.env['CERAMIC_NETWORK'] || '';
```
[the remark this answers is not in the document]
Comment (alexkeating) [reply libs/dao-data/src/Profile.ts:27]: Yes, good catch

File: libs/dao-data/src/Profile.ts [hunk 4]
```diff
@@ -25,12 +27,11 @@ type AccountProfile = {
...
 
 export default class Profile {
-  client: Core;
+  // client: Core;
```
Comment (jordanlesich) [inline libs/dao-data/src/Profile.ts:30]: comment

File: libs/dao-data/src/Profile.ts [hunk 5]
```diff
@@ -66,3 +77,59 @@ export default class Profile {
     return await provider.lookupAddress(address);
   }
 }
+
+const profilModel = {
+  definitions: {
```
Comment (jordanlesich) [inline libs/dao-data/src/Profile.ts:82]: This is a pretty massive blob of data. Would it make sense to refactor to increase readability?

**Review by skuhlmann — APPROVED** [review]

**Pull request merged**